flaml/automl/automl.py

```diff
@@ -2647,7 +2647,7 @@ def _select_estimator(self, estimator_list):
             if self._estimator_index == len(estimator_list):
                 self._estimator_index = 0
             return estimator_list[self._estimator_index]
-        min_estimated_cost, selected = np.Inf, None
+        min_estimated_cost, selected = np.inf, None
```

Comment:
What is the min `numpy` which enables this?

Reply:
flaml requires numpy>=1.17. 1.17 has it.